src/commands/instance/logs.ts

```diff
@@ -34,11 +33,9 @@ export class InstanceLogs extends Command {
 
     try {
       await this.showInstanceLogs(instance, followOption)
-    }
-    catch {
+    } catch {
```

Comment:
You shouldnt change it
Same for others

Reply:
Thats not me, thats prettier :expressionless: 
He is activated on the project and auto indent the code